xml/rmt_install.xml

```diff
@@ -219,638 +220,721 @@
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>: <literal>true</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: serviceAccount.name</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: string
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>: <literal>""</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-   </variablelist>
+   <screen>cat &lt;&lt; EOF > myvalues.yaml
+---
+app:
+scc:
+   username: UXXXXXXX
+   password: PASSXXXX
+   products_enable:
+      - SLES/15.3/x86_64
+      - sle-module-python2/15.3/x86_64
+   products_disable:
+      - sle-module-legacy/15.3/x86_64
+      - sle-module-cap-tools/15.3/x86_64
+ingress:
+enabled: true
+hosts:
+   - host: chart-example.local
+      paths:
+      - path: "/"
+         pathType: Prefix
+tls:
+- secretName: rmt-cert
+   hosts:
+   - chart-example.local
+EOF
+> helm install rmtsle oci://registry.suse.com/suse/rmt-helm -f myvalues.yaml</screen>
+
+   <sect3 xml:id="sec-rmt-helm-required-values">
+    <title>Required values</title>
+    <variablelist>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.password</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>nil</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>:
+        <link xlink:href="&sccurl;">&scc;</link> proxy password.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.username</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>nil</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>:
+        <link xlink:href="&sccurl;">&scc;</link> proxy username.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.products_enable</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: list
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>[]</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: List of products to
+        enable for mirroring
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.products_disable</term>
```

Comment:
I don't think this value is "required," is it?

Reply:
It's set as required in the chart

Reply:
Why is it required in the chart? It's kind of strange to say that you MUST disable a product.

Reply:
This is from rmt-server image `entrypoint.sh`. I agree that it should go but we can address that once we've made amendments to the docker image. Having this is necessary for documentation completeness.

Reply:
Couldn't we give it a default "empty" value?

Reply:
It has an empty list as a default value. But it's a relevant parameter for a customers' deployment atm.

Reply:
I agree with your reservation but there's no way of closing the bug without fixing the image [`entrypoint.sh`](https://build.opensuse.org/package/view_file/devel:BCI:SLE-15-SP4/rmt-server-image/entrypoint.sh?expand=1)

Reply:
But if it has a default value, it doesn't need to specified in values.yaml, or does it?

Reply:
It's a relevant variable for the customer to control. That's how I understood it.

Reply:
I don't think it's totally necessary for a first installation, since there are no "products to disable" - but I guess it doesn't hurt to leave it in.